doc/Language/intro.pod6

```diff
@@ -9,6 +9,20 @@ goals, such as being brief whilst being comprehensive, catering to professional
 developers with wide experience whilst also being accessible to newcomers to the
 language.
 
+Note the recent name change to Raku from Perl 6 has resulted in some objects
+having changed names and their former names, while still available, are
```

Comment:
I don't know why this slipped by. Probably, instead of "recent", just quote the date. Also, "objects and methods"